.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,85 @@
+name: UI bug report for Ubuntu Desktop Provision
+description: Create a report to help us improve
```

I'll make suggestions on some of the wording for clarity and conciseness. 
```suggestion
description: Report an issue to help the development
```